managed-services/transactional-email/how-to/generate-api-keys-for-api-and-smtp-sending.mdx

```diff
@@ -0,0 +1,96 @@
+---
+meta:
+  title: How to generate API keys for API and SMTP sending
+  description: This page explains how to generate API keys for API and SMTP sending
+content:
+  h1: How to generate API keys for API and SMTP sending
+  paragraph: This page explains how to generate API keys for API and SMTP sending
+dates:
+  validation: 2022-11-07
+  posted: 2022-11-07
+---
+
+This page shows you how to generate [API keys](/console/my-account/concepts/#api-key) to send [transactional emails](/managed-services/transactional-email/concepts/#transactional-email) via API and SMTP.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+</Message>
+
+## Generate an API key and send emails via the API
+
+To send your emails via the API, a valid API key and your Project's ID are required.
+
+1. From the [Scaleway console](https://console.scaleway.com), navigate to the [Credentials](/console/my-account/concepts#credentials) tab of your Project Dashboard:
+        <Lightbox src="scaleway-credentials-nav.webp" alt="" />
+2. Scroll down to the **API Key** section:
+        <Lightbox src="scaleway-api-keys.webp" alt="" />
+3. Click **Generate new API key**. The following pop-up appears:
+        <Lightbox src="scaleway-api-key-purpose.webp" alt="" />
+4. Add an **API Key purpose** (for your own reference), or leave this blank. Then click **Generate API Key**.
+
+    The two parts of your API Key (access key and secret key) are displayed. Take careful note of the secret key, as it will not be recoverable or viewable after this point.
```

Comment:
```suggestion
    The two parts of your API Key (access key and secret key) are displayed. Note the secret key carefully, as it will not be recoverable or viewable after this point.
```